extensions/cornerstone-dicom-sr/src/index.tsx

```diff
@@ -30,7 +32,27 @@ const dicomSRExtension = {
    * Only required property. Should be a unique value across all extensions.
    */
   id,
-  onModeEnter,
+
+  onModeEnter({ servicesManager }) {
+    const { toolbarService } = servicesManager.services;
+
+    toolbarService.addButtons([
```

Comment:
Perhaps add a comment indicating these are default values to be used by the toolbars, and that they can of course be changed in the mode itself.